app/i18n/cz/conf.php

```diff
@@ -85,17 +84,17 @@
 		'state_6' => 'Zobrazit oblíbené nepřečtené články',
 		'state_7' => 'Zobrazit oblíbené články',
 		'state_8' => 'Zobrazit všechny články vyjma oblíbených',
-		'state_9' => 'Zobrazit všechny přečtené články vyjma  oblíbených',
```

Comment:
What's happening here with the spaces vs tabs? Is that intentional or a side effect?

Reply:
That's a side effect. It's weird to have a tab in here as well as 2 spaces.

Reply:
I'm guessing the two spaces is just a typo, but the replacement seems a bit odd here.